korebuild.json

```diff
@@ -1,4 +1,4 @@
 {
   "$schema": "./tools/korebuild.schema.json",
-  "channel": "release/2.1"
+  "channel": "dev"
```

Comment:
acutally, this should be changed too. It's used by test.ps/sh

Reply:
Done.